make/golang.mk

```diff
@@ -52,9 +52,16 @@ golang-dep-vendor-deps: bin/dep
 # golang-godep-vendor is a target for saving dependencies with the dep tool
 # to the vendor/ directory. All nested vendor/ directories are deleted via
 # the prune command.
+# In CI, -vendor-only is used to avoid updating the lock file.
+ifndef CI
```

Comment:
@rgarcia do we prefer this approach, or doing `if` via bash like so:

https://github.com/Clever/dev-handbook/pull/103/files

Either is fine with me, just want to be consistent.

Reply:
no preference, and i'm fine using both approaches too